Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
 gem 'therubyracer', '0.10.2', :platforms => :ruby
 gem 'uglifier', '>= 1.0.3'
 gem 'will_paginate'
+gem 'omniauth-cas'
```

Comment:
I looked at the gem some more last night, and I think it looks good. So I think the only issues that need to be corrected are the Ruby styling and the customization of your own en.yml file. Check out this guide we wrote up to help you with your customizations: https://github.com/sparc-request/sparc-request/blob/master/CUSTOMIZE.md